decidim-core/lib/decidim/participatory_space_resourceable.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require "active_support/concern"
+
+module Decidim
+  # A concern with the features needed when you want a model to be able to create
+  # links from it to another resource.
```

Comment:
Can you add docs with the difference between this concern and `Decidim::Resourceable`?

Reply:
@mrcasals Do you mean creating a file inside the `/docs` folder?

Reply:
No, just improve the docs in this file!

Reply:
done